webapi/Extensions/SemanticKernelExtensions.cs

```diff
@@ -41,6 +43,11 @@ internal static class SemanticKernelExtensions
     /// </summary>
     public delegate Task RegisterSkillsWithKernel(IServiceProvider sp, IKernel kernel);
 
+    /// <summary>
+    /// Delegate to register skills with a Semantic Kernel
+    /// </summary>
+    public delegate Task RegisterSkillsWithPlanner(IServiceProvider sp, IKernel kernel);
```

Comment:
Why do we need two delegates? When would we want to only get the custom skills without the Chat Copilot ones?

Reply:
For the Planner, we don't want to register the ChatSkill with the planner